README.md

```diff
@@ -27,3 +27,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+Answer:
+A good invariant would be [0... n - 1].
```

Comment:
This is not an invariant. Please have a look at the slides and class discussion.